index.html

```diff
@@ -188,75 +188,115 @@ <h4 class="colorh4">CANOPY</h4>
+                <li>
+                    <a title="The link will take you to my linkin profile"
+                        href="https://www.linkedin.com/in/anold-mwawasi">
+                        <i class="fa fa-linkedin social"></i>
+                    </a>
                 </li>
             </ul>
+            <div class="getresume">
+                <a title="get my resume" href="#">Get my resume</a>
+            </div>
         </article>
-    </section>
+        <article id="what_i_do" class="padding">
 
+          <ul class="gridcolumn">
+            <li>
+                 <div class="mygrid">
+                    <h5>Languages</h5>
+                    <i class="fa fa-angle-down fontarrows right"></i>
+                 </div>
+                 <ul class="langcontainer">
+                    <li class="langgrid">
+                        <img class="marginleft" src="images/Ellipse 1.png" alt="">
+                        <p>javascript</p>
+                    </li>
+                    <li class="langgrid">
+                        <img src="images/Ellipse 1 (1).png" alt="">
+                        <p>HTML</p>
+                    </li>
+                    <li class="langgrid">
+                        <img src="images/Ellipse 1 (2).png" alt="">
+                        <p>CSS</p>
+                    </li>
+                 </ul>
+            </li>
+            <li class="headlang">
+              <div class="mygrid">
+                <h5>Frameworks</h5>
+                <i class="fa fa-angle-right fontarrows"></i>
+              </div>
+            </li>
+            <li class="headlang">
+               <div class="mygrid">
+                <h5>Skills</h5>
+                <i class="fa fa-angle-right fontarrows"></i>
+               </div>
+            </li>
+          </ul>
+        </article>
+
+<!---contact me article-->
+
+        <footer>
+            <section class="footer">
+              <h2>Contact me</h2>
+              <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                I'd love to help with it.
+              </p>
+              <form action="https://formspree.io/f/{form_id}" method="post">
+                <ul class="form-container">
+                 <li class="li">
+                     <label for="name">
+                         <input id="name"  maxlength="30" placeholder="Lorem ipsum" type="text" required>
+                     </label>
+                 </li>
+                 <li class="li">
+                     <label for="email">
+                         <input id="email"  type="email" placeholder="loremipsum@gmail.com" required>
+                     </label>
+                 </li>
+                 <li class="li">
+                     <textarea name="comments" id="comments" cols="30" rows="10" placeholder="write your message here..." required></textarea>
```

Comment:
- [ ] Kindly make sure that Message input has `maxlength="500"` attribute. So Kindly Fix it.